bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/IDEWorkbenchMessages.java

```diff
@@ -1147,6 +1147,12 @@ public class IDEWorkbenchMessages extends NLS {
 	public static String WorkbenchPreference_maxSimultaneousBuilds;
 	public static String WorkbenchPreference_maxSimultaneousBuildIntervalError;
 
+	public static String User;
```

Comment:
It would be better (consistent to the rest of the code) to prefix the keys with the class name which uses them